### PR TITLE
Zookeeper v3.4.8 upgradable

### PR DESCRIPTION
**Pull request by LLParse**

### CODE REVIEW

File: zookeeper/containers/0.3.0/zookeeper/README.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+ZK_ENSEMBLE_SIZE=1
+ZK_TICK_TIME=2000
+ZK_INIT_LIMIT=10
+ZK_SYNC_LIMIT=5
+ZK_MAX_CLIENT_CXNS=60
+
+```
```
Comment (LLParse) [inline zookeeper/containers/0.3.0/zookeeper/README.md]: clean up readme

File: zookeeper/0.3.0/docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
+zookeeper:
+  image: llparse/zookeeper:latest
```
Comment (cloudnautique) [inline zookeeper/0.3.0/docker-compose.yml]: We should make this rancher no?

File: zookeeper/0.3.0/rancher-compose.yml [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+      type: int
+    - variable: ZK_MAX_CLIENT_CXNS
+      description: The maximum number of accepted client connections
+      label: Max Client Connections
+      required: true
+      default: 60
+      type: int
+zookeeper:
+  scale: ${ZK_ENSEMBLE_SIZE}
+  health_check:
```
Comment (cloudnautique) [inline zookeeper/0.3.0/rancher-compose.yml:51]: lets add `retain_ip: true` That way they come back with the same IP addresses.

File: zookeeper/0.3.0/docker-compose.yml (hunk 2 above)
Comment (LLParse) [inline zookeeper/0.3.0/docker-compose.yml]: d'oh! I made the change in community-catalog, but not here. will fix

**Pull request merged**